packages/venia-ui/lib/components/MiniCart/ProductList/productList.js

```diff
@@ -0,0 +1,32 @@
+import React, { useMemo } from 'react';
+
+import LoadingIndicator from '../../LoadingIndicator';
+import Item from './item';
+import { mergeClasses } from '../../../classify';
+
+import defaultClasses from './productList.css';
+
+const ProductList = props => {
+    const { items, loading, handleRemoveItem, classes: propClasses } = props;
+    const classes = mergeClasses(defaultClasses, propClasses);
+
+    const cartItems = useMemo(() => {
+        if (!loading && items) {
```

Comment:
Did you get something figured out for when there are no items? Looks a bit weird right now:

[![Image from Gyazo](https://i.gyazo.com/a4555428984f2cdc074f14d217940708.png)](https://gyazo.com/a4555428984f2cdc074f14d217940708)

Reply:
Nice catch, It is being handled in #2511